cdap-docs/admin-manual/source/security/impersonation.rst

```diff
@@ -61,6 +61,10 @@ described in the installation section for each distribution (:ref:`Cloudera Mana
 <cloudera-hdfs-permissions>`, :ref:`Ambari <ambari-hdfs-permissions>`, 
 :ref:`MapR <mapr-hdfs-permissions>`, and :ref:`packages <packages-hdfs-permissions>`).
 
+Note that you can use the HDFS command ``hdfs groups [username ...]`` to confirm that the
+groups are set correctly, and that HBase permissions and external security services such
```

Comment:
This will not verify HBase permissions only LDAP or any other way the user groups are mapped.